ansible/playbooks/tools/dist-upgrade.yml

```diff
@@ -96,13 +96,22 @@
       stat:
         path: '{{ dist_upgrade_lockfile }}'
       register: dist_upgrade_register_lockfile
+      
+    - name: Find all apt sources.list files
+      find:
+        paths: "/etc/apt/"
+        patterns: "*.list"
+        recurse: yes
+      register: repos
```

Comment:
Sounds good, although I would use a more descriptive register name, for example `dist_upgrade_register_source_lists`. Can you update the name in this and following task?